packages/core-sdk/test/integration/dispute.test.ts

```diff
@@ -99,23 +141,87 @@ describe("Dispute Functions", () => {
     );
   });
 
-  it.skip("it should not cancel a dispute (yet)", async () => {
-    // First raise a dispute
+  it("it should not cancel a dispute (yet)", async () => {
     const raiseResponse = await clientA.dispute.raiseDispute({
       targetIpId: ipIdB,
-      cid: "QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR",
+      cid: await generateCID(),
       targetTag: "IMPROPER_REGISTRATION",
       liveness: 2592000,
       bond: 0,
       txOptions: { waitForTransaction: true },
     });
 
-    // Then you shouldnn't be able to cancel it
     expect(
       clientA.dispute.cancelDispute({
         disputeId: raiseResponse.disputeId!,
         txOptions: { waitForTransaction: true },
       }),
     ).to.be.rejected;
   });
+
+  describe("resolveDispute", () => {
+    let disputeId: bigint;
+
+    before(function (this: Mocha.Context) {
+      if (!process.env.JUDGE_PRIVATE_KEY) {
```

Comment:
:+1: great solution! I suggest we can also add some comments here on this key and why its skipped. Otherwise its not clear to other future devs reading this.

Reply:
Comments added